sinfonier-web/app/views/Layouts/darwin-control-panel-layout.html

```diff
@@ -30,7 +30,7 @@
   </div>
 </div>
 
-<script type="text/javascript" src="@{'/public/javascripts/jquery-1.11.2.min.js'}"></script>
+<script type="text/javascript" src="@{'/public/javascripts/jquery-3.1.1.min.js'}"></script>

```

Comment:
jquery datatables must be upgraded too, and all other plugins like jquery datetime
You can configure datatables js and css here:
https://datatables.net/download/index